File: app/src/main/java/com/babestudios/companieshouse/data/model/charges/InsolvencyCase.java
package com.babestudios.companieshouse.data.model.charges;


import com.google.gson.annotations.SerializedName;

public class InsolvencyCase {
	@SerializedName("case_number")
	public String caseNumber;
	@SerializedName("links")
	public InsolvencyLinks links;
	@SerializedName("transaction_id")
	public String transactionId;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/charges/InsolvencyCase.kt
package com.babestudios.companyinfouk.data.model.charges


import com.google.gson.annotations.SerializedName

class InsolvencyCase {
	@SerializedName("case_number")
	var caseNumber: String? = null
	@SerializedName("links")
	var links: InsolvencyLinks? = null
	@SerializedName("transaction_id")
	var transactionId: String? = null
}
